bin/oref0-setup.sh

```diff
@@ -682,6 +684,9 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
     if [[ ${CGM,,} =~ "g5" || ${CGM,,} =~ "g5-upload" ]]; then
         openaps use cgm config --G5
         openaps report add raw-cgm/raw-entries.json JSON cgm oref0_glucose --hours "24.0" --threshold "100" --no-raw
+    elif [[ ${CGM,,} =~ "g6" || ${CGM,,} =~ "g6-upload" ]]; then
+        openaps use cgm config --G6
+        openaps report add raw-cgm/raw-entries.json JSON cgm oref0_glucose --hours "24.0" --threshold "100" --no-raw
```

Comment:
Will need to add `set_pref_string .cgm_loop_path "$directory"` in this function to make sure `oref0-monitor-cgm.sh` runs in the right place. See: https://github.com/openaps/oref0/issues/1149 and https://github.com/openaps/oref0/pull/1211

Reply:
Thanks for fixing this bug! I ran into it a while ago and gave up on it because I couldn't track it down in the time I had. I'm closing this PR and opening a new one that both adds setup support for touchscreen receivers, fixes this bug, and makes some changes to `oref0-cron-every-minute` to make the CGM loop work with the new receivers too.